REFERENCE.md

```diff
@@ -7369,6 +7369,16 @@ X-Forwarded-Host and X-Forwarded-Server) get sent to the backend server.
 
 Default value: `undef`
 
+##### `proxy_requests`
+
+Data type: `Any`
```

Comment:
You've added `proxy_requests` as a boolean here https://github.com/puppetlabs/puppetlabs-apache/pull/1935/files#diff-c918e8884948c0fae251b33071374150R1708 but document it it as `Data type: 'Ant'` here. Please correct this, thank you.